app/helpers/ncr/work_orders_helper.rb

```diff
@@ -1,9 +1,8 @@
 module Ncr
   module WorkOrdersHelper
     def approver_options
-      # @todo should this list be limited by client/something else?
       # @todo is there a better order? maybe by current_user's use?
-      User.order(:email_address).pluck(:email_address)
+      User.where(client_slug: 'ncr').order(:email_address).pluck(:email_address)
```

Comment:
I tend to have the same thinking as you. Per this conversation with Aidan, however, I'm trying to keep things out of the model that are just view-based extensions: https://github.com/18F/C2/pull/681#discussion_r41535674